### PR TITLE
PLT-7: Refactoring frontend (chunk 6)

### PR DESCRIPTION
**Pull request by enahum**
- Authorize
- Signup Team Confirm
- Footer

### CODE REVIEW

File: web/react/components/signup_team_confirm.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+/**
+ * Created by enahum on 1/29/16.
+ */
```
Comment (jwilander) [inline web/react/components/signup_team_confirm.jsx]: Can you remove this? It'll be tracked in GitHub

File: web/react/pages/signup_team_confirm.jsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
+/**
+ * Created by enahum on 1/29/16.
+ */
```
Comment (jwilander) [inline web/react/pages/signup_team_confirm.jsx]: Here too

Comment (enahum) [inline web/react/pages/signup_team_confirm.jsx]: done!

**Comment by jwilander** [comment]
Thanks! +1

**Comment by crspeller** [comment]
+1

**Pull request merged**